js/components/post-select/modal.js

```diff
@@ -103,23 +103,21 @@ class PostSelectModal extends React.Component {
 				</div>
 				<div className="media-frame-toolbar">
 					<div className="media-toolbar">
+						<Button
+							isPrimary={true}
+							isLarge={true}
```

Comment:
These should usually be alphabetised and with spaces inside the `{}`, but matches surrounding style.

Reply:
Regarding the spaces inside the `{}` - The HM eslint rules don't seem to have an opinion either way on this one. I think I'd prefer the spaces - but would be good to make sure this is enforced/fixed by ESLint. I'm happy to merge for now because - as you say - matches surrounding style.

Reply:
Added a PR here: https://github.com/humanmade/coding-standards/pull/38